R/page.R

```diff
@@ -161,16 +252,59 @@ page_sidebar <- function(..., sidebar = NULL, title = NULL, fillable = TRUE, fil
 }
 
 
-#' @rdname page
-#' @inheritParams navset_bar
-#' @seealso [shiny::navbarPage()]
+#' Multi-page app with a top navigation bar
+#'
+#' @description
+#' `page_navbar()` provides a full-screen app with a navigation bar that behaves
+#' like a multi-page app (but is really a single-page app).
```

Comment:
I think there is some potential for folks to be confused by what you mean by `full-screen` and `multi-page`/`single-page`. Maybe we could take some inspiration from `shiny::navbarPage()`'s description:

> Create a page that contains a top level navigation bar that can be used to toggle a set of [nav_panel()] elements.

Reply:
I just copied that directly, but I added a little more context:

```
Create a page that contains a top level navigation bar that can be used to
toggle a set of [nav_panel()] elements. Use this page layout to create the
effect of a multi-page app, where your app's content is broken up into
multiple "pages" that can be navigated to via the top navigation bar.
```

I think most people's mental model will be to think of `page_navbar()` as a multi-page app, but we also want some hints in case some one with a very technical understanding of multi-page apps lands on the `page_navbar()` docs.